Sources/SnapshotSafeView/Core/HiddenContainerRecognizer.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  HiddenContainerRecognizer.swift
+//  
+//
+//  Created by Князьков Илья on 23.07.2022.
+//
+
+import UIKit
+
+/// Recognize view, which can be hidden before system screenshot event did triggered, depend on `iOS vesrion`
```

Comment:
typo

Reply:
Thanks :), fixed. in https://github.com/Stampoo/SnapshotSafeView/pull/5